tests/translator/input/api_with_method_aws_iam_auth.yaml

```diff
@@ -27,7 +27,7 @@ Resources:
             Method: post
             Auth:
               Authorizer: AWS_IAM
-              InvokeRole: rn:aws:iam::123:role/AUTH_AWS_IAM
+              InvokeRole: !Sub arn:${AWS::Partition}:iam::123:role/AUTH_AWS_IAM
```

Comment:
Why this change?

Reply:
The arn was incorrect `rn:aws:iam...`.

Reply:
Okay, so I'm assuming it transformed correctly. Transform tests should ideally never change, so we can skip making these changes as well (maybe make a note for later cleanup) for easier review (e.g. I'm not sure whether the logical ID change is due to this or the indentation change).

Reply:
The transform is correct, and the id change is due to indentation change I believe. It's when I make the indentation change, I found the arn seems off. And there are two occurrence of wrong arn format.